libpod/lock/shm/shm_lock.go

```diff
@@ -29,7 +29,7 @@ var (
 
 // SHMLocks is a struct enabling POSIX semaphore locking in a shared memory
 // segment.
-type SHMLocks struct {
+type SHMLocks struct { //nolint:revive
```

Comment:
(A drive-by comment, I didn’t read the full PR.)

Consider documenting what exactly is being disabled.